sql/codegen.go

```diff
@@ -276,59 +259,98 @@ classifier = {{.Estimator}}(
 	{{end}}
 
 {{if .Train}}
-def train_input_fn(features, labels, batch_size):
-    dataset = tf.data.Dataset.from_tensor_slices((dict(features), labels))
-    dataset = dataset.shuffle(1000).repeat().batch(batch_size)
-    return dataset
+def train_input_fn(batch_size):
```

Comment:
As `train_input_fn` and `eval_input_fn` becomes more and more complicated, shall we move it to `sql/python/sqlflow_submitter/input_fn.py` and create corresponding tests?

Reply:
I would like to keep it in here because these functions are using a lot of global variables defined above, moving it to another file means we must pass those variables as parameters. These parameters including:

- `feature_column_names`
- `column_name_to_type`
- `"""{{.StandardSelect}}"""`
- `"{{.Y.Name}}"`
- `driver`
- `conn`